docs/pages/Configuration.md

```diff
@@ -691,6 +694,7 @@ Unless otherwise noted in the table below, UI5 Tooling modules are backward comp
 
 Version | UI5 CLI Release
 --- | ---
+**3.0** | v3.0.0+
```

Comment:
As discussed in the planning, add a note that v3 is currently in beta. Ideally linking to the migration page.